src/actions/track-chair-actions.js

```diff
@@ -375,3 +385,135 @@ export const reorderProgressFlags =
       dispatch(stopLoading());
     });
   };
+
+/** ********************************************************************************************************* */
+/*                          TEAM LISTS                                                                        */
+/** ********************************************************************************************************* */
+
+export const getSelectionPlans = () => async (dispatch, getState) => {
+  const { currentSummitState } = getState();
+  const accessToken = await getAccessTokenSafely();
+  const { currentSummit } = currentSummitState;
+
+  dispatch(startLoading());
+
+  const params = {
+    page: 1,
+    per_page: 100,
+    access_token: accessToken,
+    expand: "track_groups,track_groups.tracks"
+  };
+
+  return getRequest(
+    null,
+    createAction(RECEIVE_SELECTION_PLANS),
+    `${window.API_BASE_URL}/api/v1/summits/${currentSummit.id}/selection-plans`,
+    authErrorHandler
+  )(params)(dispatch).then(() => {
+    dispatch(stopLoading());
+  });
+};
+
+export const setSelectionPlan = (target, selectionPlanId) => (dispatch) => {
+  const action = target === "source" ? SET_SOURCE_SEL_PLAN : SET_TEAM_SEL_PLAN;
+  dispatch(createAction(action)({ selectionPlanId }));
+};
+
+export const getSourceList =
+  (selectionPlanId, trackId, searchTerm = "", page = 1) =>
+  async (dispatch, getState) => {
+    const { currentSummitState } = getState();
+    const accessToken = await getAccessTokenSafely();
+    const { currentSummit } = currentSummitState;
+    const filter = [
+      `selection_plan_id==${selectionPlanId}`,
+      `track_id==${trackId}`,
+      "status==Received",
+      "progress==5"
+    ];
+
+    dispatch(startLoading());
+
+    if (searchTerm) {
+      const escapedTerm = escapeFilterValue(searchTerm);
+      if (Number.isInteger(parseInt(escapedTerm, 10))) {
+        filter.push(
+          `id==${escapedTerm},title=@${escapedTerm},abstract=@${escapedTerm},speaker=@${escapedTerm}`
+        );
+      } else {
+        filter.push(
+          `title=@${escapedTerm},abstract=@${escapedTerm},speaker=@${escapedTerm}`
+        );
+      }
+    }
+
+    const params = {
```

Comment:
could we restrict requested data to minimum needed ( fields, relations and expands ) ?